collection_modules/btle_beacon/devices/bluegiga/bluegigaDevice.py

```diff
@@ -2,21 +2,23 @@
 # https://github.com/jrowberg/bglib/blob/master/Python/Examples/bled112_scanner.py
 #
 #
-from simplesensor.collection_modules.btle_beacon.libs import BGLib
+from .bglib import BGLib
```

Comment:
this was also addressed in the last PR.  your working with some old files.

Reply:
"